### PR TITLE
2.03 changes to codelist management

### PR DESCRIPTION
**Pull request by amy-silcock**
See https://github.com/IATI/IATI-Extra-Documentation/pull/549

### CODE REVIEW

**Review by PetyaKangalova — APPROVED** [review]
We have discussed the proposed changes with @amy-silcock and the amendments look good. @andylolz  thanks for flagging this.

File: en/codelists-guides/codelist-management.rst [hunk 1]
```diff
@@ -43,7 +42,7 @@ Examples of Core Codelists include: Transaction Type, Organisation Type.
 Non-Embedded Codelists - Non-Core
 ----------------------
 
-Non-Core Codelists contain conforming values for a particular **integer** version of the IATI Standard, the contents of which are managed by the IATI Technical Team. Non-Core Codelists contain Codes that qualify data, not processes.
+Non-Core Codelists contain values which do not conform to a particular version of the IATI Standard. The contents of the codelist which are managed by the IATI Technical Team. Non-Core Codelists contain codes that qualify data, not processes.
```
Comment (andylolz) [inline en/codelists-guides/codelist-management.rst:45]: > The contents of the codelist which are managed by the IATI Technical Team.

^^ This isn’t a complete sentence.

> Non-Core Codelists contain values which do not conform to a particular version of the IATI Standard.

^^ I wonder if there’s a clearer way to say this?

File: en/codelists-guides/codelist-management.rst [hunk 2]
```diff
@@ -64,7 +63,7 @@ Examples of Non-Core Codelists include: Sector Vocabulary, Condition Type, Resul
 Non-Embedded Codelists - Replicated
 ----------------------
 
-Replicated Codelists contain values for the IATI Standard. The contents are generally managed by a third-party organisation, in exceptional circumstances an additional code can be added. The IATI Technical Team maintain a corresponding copy in a document that conforms to the Codelist Schema.
+Replicated Codelists contain values for the IATI Standard. The contents are generally managed by a third-party organisation. In exceptional circumstances an additional code can be added. The IATI Technical Team maintains a corresponding copy in a document that conforms to the Codelist Schema.
```
Comment (andylolz) [inline en/codelists-guides/codelist-management.rst:66]: > Replicated Codelists contain values for the IATI Standard.

^^ I think this is true of all codelist types.

File: en/codelists-guides/codelist-management.rst (hunk 1 above)
Comment (PetyaKangalova) [reply en/codelists-guides/codelist-management.rst:45]: Maybe we can say Non-Core Codelist values are not version specific?

Comment (andylolz) [reply en/codelists-guides/codelist-management.rst:45]: Sounds good to me.

**Pull request merged**